lib/tests/test_signing.rs

```diff
@@ -95,7 +90,7 @@ fn keep_on_rewrite(backend: TestRepoBackend) {
     let rewritten = mut_repo.rewrite_commit(&settings, &commit).write().unwrap();
 
     let commit = repo.store().get_commit(rewritten.id()).unwrap();
-    assert_eq!(commit.verification().unwrap(), good_verification());
+    assert_eq!(format!("{:?}", commit.verification()), GOOD_VERIFICATION);
```

Comment:
nit: since we're doing string-based assertion, maybe we can use `insta::assert_debug_snapshot!(commit.verification(), @"")` instead?

Reply:
I tried but I don't know how to do this without inlining the same snapshot at all the places this is used. Is there a way to do this while keeping the `GOOD_VERIFICATION` const defined at the top to be shared?

Reply:
Maybe I misunderstood why you made it a string. (I just thought it would be tedious to update the good `Verification { .. }` struct manually.)

To get around the private `backend` field, I would probably make it assert each field one by one. It might be extracted to `is_good_verification()` helper.